geometry_msgs/Pose pose
float32 duration
bool use_orientation
bool use_position
string task_name